pkg/standalone/run.go

```diff
@@ -374,3 +348,13 @@ func Run(config *RunConfig) (*RunOutput, error) {
 		DaprGRPCPort: daprGRPCPort,
 	}, nil
 }
+
+func getComponentsPath(config *RunConfig) (string, error) {
+	if config.ComponentsPath == "" {
+		componentsPath := getDefaultComponentsFolder()
+		return componentsPath, nil
+	}
+
+	_, err := os.Stat(config.ComponentsPath)
+	return config.ComponentsPath, err
```

Comment:
is the expected behavior that the `config.ComponentsPath` exists previously and not created on the fly as was the previous behavior?

Reply:
The path was created during dapr run before. Now it will be created at dapr init time